src/views/static/Resources.vue

```diff
@@ -69,12 +69,12 @@ export default {
             description: 'Official website',
             color: '#4CA0FF'
           },
-          {
-            url: 'https://aegee.blogactiv.eu/',
-            title: 'External Blog',
-            description: 'Blog hosted on Blogactiv platform',
-            color: '#4CA0FF'
-          },
+          // {
```

Comment:
you can also remove it, it's anyway in git history

Reply:
True, fixed